packages/ui/v2/core/apps/AppCard.tsx

```diff
@@ -16,8 +16,10 @@ interface AppCardProps {
 export default function AppCard({ app, credentials }: AppCardProps) {
   const { t } = useLocale();
   const mutation = useAddAppMutation(null, {
-    onSuccess: () => {
-      showToast(t("app_successfully_installed"), "success");
+    onSuccess: (data: any) => {
+      if (data.showToast !== undefined && data.showToast) {
```

Comment:
This is to fix the false positive "App successfully installed" toast message
cc: @hariombalhara